lib/ansible/modules/system/pamd.py

```diff
@@ -384,7 +385,7 @@ def write(self):
             self.fname = self.path + "/" + self.name
         # If the file is a symbollic link, we'll write to the source.
         pamd_file = os.path.realpath(self.fname)
-        temp_file = "/tmp/" + self.name + "_" + time.strftime("%y%m%d%H%M%S")
+        temp_file = os.path.join(tempfile.gettempdir(), self.name + "_" + time.strftime("%y%m%d%H%M%S"))
```

Comment:
The base module object has a tempdir property (`module.tempdir`), which means you can reuse the tempdir the module itself used

Reply:
Thanks, but I scanned all of the other modules and didn't see any use of this.  Where is this defined?  Is it part of AnsibleModule?  Can you point me to a link to where it is?

Reply:
https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/basic.py#L40

^ list of attributes AnsibleModule sets based on internal vars from controller

it is a recent addition and I've just only started updating modules to use it.